### ISSUE TITLE
Undefined behavior

### ISSUE THREAD

**Issue by rgrmra** [vinicius-f-pereira/philosophers#1]
Your `ft_atoi_recursive` has an undefined behavior when you try to pass numbers with minus in the middle of the string, like: 1-1-1-1 or 111-1.

https://github.com/0bvim/philosophers/blob/38d6cba50a236075e20f8f0c3f955b2c00377e06/philo/src/utils/ft_atoi.c#L26

**Comment by 0bvim** [comment]
> Your `ft_atoi_recursive` has an undefined behavior when you try to pass numbers with minus in the middle of the string, like: 1-1-1-1 or 111-1.
> 
> https://github.com/0bvim/philosophers/blob/38d6cba50a236075e20f8f0c3f955b2c00377e06/philo/src/utils/ft_atoi.c#L26

I can't reproduce the error...
I tried
```bash
./philo 5 800 111-1 200
./philo 5 800 1-1-1-1 200
```

**Comment by rgrmra** [comment]
No, your program is working properly. The error is exclusive from `ft_atoi_recursive`. I'm just warning you that it has an undefined behavior in some cases and if you want to use it in a future program.
